spec/outputs/elasticsearch_spec.rb

```diff
@@ -486,8 +486,13 @@ def mock_actions_with_response(*resp)
         .any_instance.stub(:bulk).and_return(*resp)
       LogStash::Outputs::Elasticsearch::Protocols::NodeClient
         .any_instance.stub(:bulk).and_return(*resp)
-      LogStash::Outputs::Elasticsearch::Protocols::TransportClient
-        .any_instance.stub(:bulk).and_return(*resp)
+    end
+
+    def unmock_actions()
```

Comment:
Until doing a more detailed knowledge finding, the best resource about scoping I could find is https://github.com/rspec/rspec-mocks/blob/master/features/basics/scope.feature